packages/react-composites/tests/browser/call/hermetic/ParticipantPane.test.ts

```diff
@@ -86,4 +87,41 @@ test.describe('Participant pane tests', async () => {
 
     expect(await stableScreenshot(page)).toMatchSnapshot(`PSTN-call-screen-with-dialpad.png`);
   });
+
+  test.only('callee participant is displayed with connection state', async ({ page, serverUrl }) => {
+    test.skip(isTestProfileStableFlavor());
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
+    paul.state = 'Connecting';
+    const participants = [paul];
+    const initialState = defaultMockCallAdapterState(participants);
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    await pageClick(page, dataUiId('call-composite-participants-button'));
+    await waitForSelector(page, dataUiId('call-composite-people-pane'));
+    await hidePiPiP(page);
+    expect(await stableScreenshot(page)).toMatchSnapshot('PSTN-participant-pane-callee-name.png');
+  });
+
+  test.only('callee participant name and connection state are truncated', async ({ page, serverUrl }) => {
+    test.skip(isTestProfileStableFlavor());
+    const longPaul = defaultMockRemoteParticipant(
+      'I have a really really really really long name. Trust me you dont wanna know.'
+    );
+    longPaul.state = 'Connecting';
+    const participants = [longPaul];
+    const initialState = defaultMockCallAdapterState(participants);
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    await pageClick(page, dataUiId('call-composite-participants-button'));
+    await waitForSelector(page, dataUiId('call-composite-people-pane'));
+    await hidePiPiP(page);
+    const participantStringId = dataUiId('participant-item-state-string');
+    await page.evaluate((participantStringId) => {
+      const el = document.querySelector(participantStringId);
+      if (el) {
```

Comment:
This is a bit hacky. Is there no way we can test actually need to truncate the connection state string?

How about zooming in so all text is larger and hence needs to be truncated?

Reply:
Zooming in changes the entire UI and might not reveal inconsistencies we end up introducing in the future.
Not sure how this is hacky, we do something similar to stub out timestamps as well.
Ultimately, what we are testing here is the css